docs/CONTRIBUTOR.md

```diff
@@ -2,10 +2,55 @@
 
```

Comment:
We should add a short introduction of what's the objective of the CONTRIBUTOR file.
Like "The aim is to guide news users to get started working with the project."